booking-engine-widget/use-cases/disable-widget-caching.md

```diff
@@ -0,0 +1,61 @@
+# Do not use a booking engine script cached by your server
+
+In the dynamic world of web development, ensuring your site's features remain up-to-date without manual intervention is crucial for providing a seamless user experience. One such feature critical to keeping your site current is the script `distributor.min.js` provided by Mews. This script is a reverse proxy for always delivering the latest version of the distributor file. However, when this script is cached, it can lead to serving an outdated version, depriving users of new features and enhancements, may also cause the booking engine to malfunction.
```

Comment:
Hey, the English is pretty good here, did you get AI assistance? 😉 
```suggestion
In the dynamic world of web development, ensuring your site's features remain up-to-date without manual intervention is crucial for providing a seamless user experience. One such feature critical to keeping your site current is the script `distributor.min.js` provided by **Mews**. This script is a reverse proxy for always delivering the latest version of the distributor file. However, when this script is cached, it can lead to serving an outdated version, depriving users of new features and enhancements. It may also cause the booking engine to malfunction.
```

Reply:
Yes, it was with a little help of our GPT friends :-D